README.md

```diff
@@ -100,9 +100,11 @@ The `token` and `site` vars must be given to testing scripts either using
 config.json file into `test/` folder like bellow:
 
 ```json
-{
-	"site": "<site-id>",
-	"token": "<token>"
+{ "development": {
+	"site": "site.ID or domain",
+	"token": "Your Token",
+	"skipRestrictedEndpoints": true, // some endpoints require special permissions to access
```

Comment:
The invalid syntax in JSON is pretty eye-piercing in the docs :smile: Also, comma after the last property will likely similarly break upon being loaded.